abdiff/cli.py

```diff
@@ -148,7 +149,12 @@ def init_job(
     help="Message to describe Run.",
     default="Not provided.",
 )
-def run_diff(job_directory: str, input_files: str, message: str) -> None:
+@click.option(
+    "--download-files", is_flag=True, help="Pass to skip download of extract files"
```

Comment:
Does the help message need updating?  I would think it's something like,

> "Pass to download input files from AWS S3 to a local Minio S3 server for Transmogrifier to use."

Reply:
Updated.